src/Helper/ContentProxy.php

```diff
@@ -102,7 +102,7 @@ public function updateLanguage(Entry $entry, $value)
         );
 
         if (0 === \count($errors)) {
-            $entry->setLanguage($value);
+            $entry->setLanguage(\Locale::canonicalize($value));
```

Comment:
Why is this now needed while it wasn't before?

Reply:
@yguedidi This is not mandatory per-se but in the case of the new address `$value`  would be "pt_br". I though using `Locale::canonicalize()` here would unify languages